fairmq/shmem/Manager.h

```diff
@@ -148,7 +148,7 @@ class Manager
     void Resume() { fInterrupted.store(false); }
     void Reset()
     {
-        if (fMsgCounter.load() != 0) {
+        if (fMsgCounter.load(std::memory_order_acquire) != 0) {
```

Comment:
I don't know the context where `Reset` is used. But from the immediate context the load could also be relaxed, no?

Reply:
`Reset` should be called only once on process exit, after all `inc/dec` operations have been completed. Default would probably also be fine.

Reply:
Right, if it's not performance critical at all, just leave it at sequentially consistent.